content/tutorials/_includes/upload_api.md

```diff
@@ -3,26 +3,25 @@ The option is also useful if you want to automate this step.
 
```

Comment:
We should remove this whole section, which has nothing to do there. The goal of these tutorials is to have as few steps as possible to have a working configuration.